src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+type Props = {
+  todo: PreparedTodo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <div className="todo-info">
+    {todo.user && <UserInfo user={todo.user} />}
+    <span className="todo-info__task">Task:</span>
+    {' '}
```

Comment:
better also use css styling for this 

```suggestion
    <span className="todo-info__task">Task:</span>
```